src/app/components/song/song.ui

```diff
@@ -63,13 +63,7 @@
         <property name="valign">center</property>
         <property name="relief">none</property>
         <property name="always-show-image">True</property>
-        <child>
-          <object class="GtkImage">
-            <property name="visible">True</property>
-            <property name="can-focus">False</property>
-            <property name="stock">gtk-media-play</property>
-          </object>
-        </child>
+        <property name="icon-name">media-playback-start-symbolic</property>
```

Comment:
I think we need to keep the GtkImage child:
`Gtk-ERROR **: 21:33:04.295: failed to add UI: .:156:53 Invalid property: GtkButton.icon-name`

Reply:
Ah, sorry, that's for GTK4, fixed